assets/js/phoenix_live_view/js.js

```diff
@@ -95,21 +95,37 @@ let JS = {
 
   exec_focus(e, eventType, phxEvent, view, sourceEl, el){
     ARIA.attemptFocus(el)
+    // in case the JS.focus command is in a JS.show/hide/toggle chain, in the show case
+    // we need to wait for the element to actually be visible, which takes two animation frames
```

Comment:
This comment and the ones below are slightly misleading, as `requestAnimationFrame` invokes the supplied callback [**_before_** the next animation frame](https://developer.mozilla.org/en-US/docs/Web/API/Window/requestAnimationFrame), so the "doubleRaf" technique used here is ensuring your code runs after a single animation frame has passed.